include/TrustWalletCore/TWHarmonyChainID.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+#include "TWBase.h"
+
+TW_EXTERN_C_BEGIN
+
+/// Chain identifier for Harmony blockchain.
+TW_EXPORT_ENUM(uint32_t)
+enum TWHarmonyChainID {
+    TWHarmonyChainIDMainNet = 0x1,
+    TWHarmonyChainIDBetaNet = 0xc8,
```

Comment:
can you remove all changes besides mainet?

Reply:
actually we don't need these properties, networking layer would be responsible for fetching from them the node itself, can you remove them from here? feel free to hardcode them in tests only if needed